task_2/index.js

```diff
@@ -0,0 +1,76 @@
+const city = document.getElementById("city");
+const companyName = document.getElementById("company-name");
+const email = document.getElementById("email");
+const password = document.getElementById("password");
+const passwordCheck = document.getElementById("password-check");
+const regularExpressionForEmail = /([a-zA-Z0-9._-]+@[a-zA-Z0-9._-]+\.[a-zA-Z0-9_-]+)/;
+const regularExpressionForPassword = /^(?=.*[0-9])(?=.*[!@#$%^&*])[a-zA-Z0-9!@#$%^&*]{8,}$/;
+
+checkValidate = (e) => {
```

Comment:
Add a new function to hideErrors instead of repeating if else 
and call it each time on submit form